bika/lims/browser/workflow/__init__.py

```diff
@@ -163,6 +164,8 @@ def do_action(self, action, objects):
         """Performs the workflow transition passed in and returns the list of
         objects that have been successfully transitioned
         """
+
+        start = time.time()
```

Comment:
Should this be removed?

Reply:
I'd leave this. Is always interesting to know how much time a given action takes